src/systems/eu/implementations/react/components/select/src/Select.jsx

```diff
@@ -6,6 +6,7 @@ import classnames from 'classnames';
 import Icon from '@ecl/eu-react-component-icon';
 
 const Select = ({
+  defaultValue,
```

Comment:
Not necessary, native prop `value` which is closer to non-react implementation will work as well https://reactjs.org/docs/forms.html

Try setting `value="4"` on single select in the storybook, it works. `props` are spread over, no need to add anything to achieve this.

Reply:
Btw, our implementation can be considered as uncontrolled because we don't do state management.